src/main.cpp

```diff
@@ -1,9 +1,15 @@
 #include <QApplication>
 #include <QIcon>
 #include <QFile>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJsonArray>
+#include "config.h"
 
 #include "mainview.h"
 
+QJsonDocument defaultConfig();
```

Comment:
nit: Why is this declared here when we also declare this in the `config.h`?

Reply:
Accidental leftover, will be removed.